virttest/utils_disk.py

```diff
@@ -135,7 +135,7 @@ def mount(src, dst, fstype=None, options=None, verbose=False, session=None):
     return process.system(cmd, verbose=verbose) == 0
 
 
-def umount(src, dst, fstype=None, verbose=False, session=None):
```

Comment:
Hi @hellohellenmao 
I suggest that you could customize the argument for the test case, rather than change the default here since this change will affect the previous behaviors and we need to involve more reviewers to evaluate it.
In order to reduce the influence and satisfy your requirements, I suggest specifying `verbose=True` in your scenario.

Reply:
Ha, make sense. Will close this PR, thanks.